controller/controlplane/controller_reconciler_utils.go

```diff
@@ -435,6 +435,10 @@ func (r *Reconciler) ensureAdminMTLSCertificateSecret(
 			Name:      r.ClusterCASecretName,
 		},
 		usages,
+		secrets.KeyConfig{
+			Type: r.ClusterCAKeyConfig.Type,
+			Size: r.ClusterCAKeyConfig.Size,
+		},
```

Comment:
Couldn't it be passed as is? 
```suggestion
		r.ClusterCAKeyConfig,
```

Reply:
🤦 ofc. Thanks for catching this.